VultisigApp/VultisigApp/iOS/View/ChainDetailView+iOS.swift

```diff
@@ -60,6 +64,14 @@ extension ChainDetailView {
         }
     }
     
+    var addCustomTokenButton: some View {
+        Button {
+            sheetType = .customToken
```

Comment:
Also , can we also hardcode some of the most common tokens on Solana into our token store?

Reply:
Yes, def we can.